pkg/interface/link/src/js/components/lib/link-detail-preview.js

```diff
@@ -99,12 +99,12 @@ export class LinkPreview extends Component {
           className={"w-100 tc " + (ytMatch ? "embed-container" : "")}>
           {embed}
         </div>
-        <div className="flex flex-column ml2 pt6">
+        <div className="flex flex-column ml2 pt6 flex-auto">
           <a href={props.url} className="w-100 flex" target="_blank">
-            <p className="f8 truncate" style={{overflow: "visible"}}>
+            <p className="f8 truncate flex-auto">
```

Comment:
The `overflow: "visible"` was there so that text didn't get cut off on the y-axis, but I guess the flex also accounts for that?

Reply:
It doesn't look cut off in the screenshot. I am pretty sure it does do the same thing, but feel free to check.